README.md

```diff
@@ -4,19 +4,19 @@ The types in this package are pydantic types for Ethereum inspired from [eth-typ
 
 ## Hash
 
-`HashBytes{n}` and `HashStr{n}` are good types to use when your hex values are sized.
+`Bytes{n}` and `String{n}` are good types to use when your hex values are sized.
 Both types serialize to `string` in the JSON schema.
-Use `HashBytes` types when you want types to serialize to bytes in the Pydantic core schema and `HashStr` types when you want to serialize to `str` in the core Pydantic schema.
+Use `Bytes` types when you want types to serialize to bytes in the Pydantic core schema and `String` types when you want to serialize to `str` in the core Pydantic schema.
 
 ```python
 from pydantic import BaseModel
 
-from eth_pydantic_types import HashBytes32, HashStr20
```

Comment:
Can you include an example of a non-predefine type that is bound?

Like how do I use BoundHexBytes sizes other than 32 and 20?